assignments/callbacks.js

```diff
@@ -25,31 +25,90 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 */
 
 
+ 
+
 function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
+  return cb();
 }
+console.log(getLength(items, function () {
+  return items.length;
+}) );
+ 
```

Comment:
Good evening.  I just wanted to show you the solution for this problem.  Can you go and refactor the rest of your answers to match this one.

```sh
function getLength(arr, cb) {
    return cb(arr.length);
}
getLength(items, function(length) {
   console.log(length);
}
```
If you have any further questions, feel free to reach out to your Team Lead or myself during the vacation week.  

Thanks,

Donald Whitely, TL, WEBPT8